commune_app/all_models/votes.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from .chores import Chore
+from .users import User
+
+
+class Vote(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    chore = models.ForeignKey(Chore, on_delete=models.CASCADE)
+    approve = models.BooleanField(default=False)
+
+    # creates a new vote while checking if the related Chore can be passed or not
+    @staticmethod
+    def create_new_vote(voting_user, voted_chore, vote_bool):
+        Vote(user=voting_user, chore=voted_chore, approve=vote_bool).save()
+        # temporarily compared the number of votes to 3 until we get a method in commune model
+        # that returns the number of members in the commune
+        if len(Vote.objects.filter(chore=voted_chore)) == 3:
```

Comment:
This needs to pull the from the commune object to get the amount of users

Reply:
yes i will replace the 3 with the number of members in the commune , 3 is temporary